### PR TITLE
add python solution to cses-1641.mdx

### PR DESCRIPTION
**Pull request by 876pol**
_If any of the below doesn't apply to this Pull Request, mark the checkbox as done._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which includes the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making them.

### CODE REVIEW

File: solutions/cses-1641.mdx [hunk 1]
```diff
@@ -132,5 +132,42 @@ public class SumOfThreeValues {
...
+<PySection>
+```python
+# read input
+n, x = map(int, input().split())
+a = list(map(int, input().split()))
+
+# store the input in tuples that hold values and indices
+p = [(a[i], i + 1) for i in range(n)]
+
+# sort in ascending order
```
Comment (SansPapyrus683) [inline solutions/cses-1641.mdx:145]: these 3 comments state the obvious

File: solutions/cses-1641.mdx [hunk 2]
```diff
@@ -132,5 +132,42 @@ public class SumOfThreeValues {
...
+p.sort()
+
+for i in range(n):
+    left = 0
+    right = n - 1
+
+    while left < right:
+        target = x - p[i][0]
+
+        # if the values sum to the target and aren't from the same index, we have found the answer
```
Comment (SansPapyrus683) [inline solutions/cses-1641.mdx:155]: over 80 chars
```py
"""
do multiline comments
like this
```

File: solutions/cses-1641.mdx [hunk 3]
```diff
@@ -132,5 +132,42 @@ public class SumOfThreeValues {
...
+
+        # increase left if the sum is lower
+        elif p[left][0] + p[right][0] < target:
+            left += 1
+
+        # otherwise decrease right
+        else:
+            right -= 1
+
+# print impossible if no answer has been found
```
Comment (SansPapyrus683) [inline solutions/cses-1641.mdx:168]:
```suggestion
```

**Review by SansPapyrus683 — CHANGES_REQUESTED** [review]

**Review by envyaims — APPROVED** [review]
seems great!

**Review by ryanchou-dev — APPROVED** [review]
address sans' comments and it should be good

File: solutions/cses-1641.mdx [hunk 4]
```diff
@@ -132,5 +132,42 @@ public class SumOfThreeValues {
...
+		# increase left if the sum is lower
+		elif p[left][0] + p[right][0] < target:
+			left += 1
+
+		# otherwise decrease right
+		else:
+			right -= 1
+
+print("IMPOSSIBLE")
+
```
Comment (SansPapyrus683) [inline solutions/cses-1641.mdx:169]:
```suggestion
```

**Review by SansPapyrus683 — CHANGES_REQUESTED** [review]
one last thing

**Review by SansPapyrus683 — APPROVED** [review]

**Pull request merged**